perma_web/frontend/components/CreateLink.vue

```diff
@@ -0,0 +1,149 @@
+<script setup>
+import { ref, watch, onBeforeUnmount } from 'vue'
+import { globalStore } from '../stores/globalStore'
+import { getCookie } from '../../static/js/helpers/general.helpers'
+// import * as spinner from 'spin.js'
+
+const userLink = ref('')
+const userLinkGUID = ref('')
+const userLinkProgressBar = ref(0)
+
+const readyStates = ["ready", "urlError", "captureError"]
+const isReady = readyStates.includes(globalStore.captureStatus)
+
```

Comment:
While this worked initially in testing, I think I should make this a computed property, for example: 

```
const isReady = computed(() => { readyStates.includes(globalStore.captureStatus) })
```

Reply:
For sure! Fixed in 65b58f8